astropy/units/tests/test_equivalencies.py

```diff
@@ -90,7 +90,7 @@ def test_doppler_wavelength_0(function):
 @pytest.mark.parametrize(('function'), doppler_functions)
 def test_doppler_energy_0(function):
     rest = 105.01 * u.GHz
-    q1 = 0.0004342864612223407 * u.eV
+    q1 = 0.0004342864648539744 * u.eV
```

Comment:
Simple solution of trying to apply ratio of `e` from different versions did not work, as equivalency seems pretty complicated. So in the end, I simply re-calculated `eV` value from the answer.